i18n/de/docusaurus-plugin-content-docs/current/toolbox/accessibility_indicators/closest_average.md

```diff
@@ -80,27 +77,29 @@ If you would like to perform analyses beyond this geofence, feel free to [contac
 
 <TabItem value="walk" label="Walk" default className="tabItemBox">
 
-#### Walk
+##### Gehen
```

Comment:
Since we're referring to a mode here, should this be "Zu Fuß" like in other places?